docs/superpowers/plans/2026-06-08-claims-immutable-inputs.md

```
@@ -0,0 +1,500 @@
+- [ ] **Step 4: Manual smoke** on `/claims` + `/portfolio` (:5180): all months display, claim still works (orders resolved entirely from static data; proofs built on click).
+
+- [ ] **Step 5: Commit**
+
+```bash
+git add src/lib/services/ClaimsService.ts src/lib/data/repositories/claimsRepository.ts src/lib/data/repositories/index.ts
+git commit -m "perf(claims): drop subgraph order lookup; all orders resolved statically"
+```
+
+---
+
+## Done criteria
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)